package.json

```diff
@@ -48,6 +48,8 @@
   "devDependencies": {
     "benchmark": "^2.1.4",
     "bluebird": "^3.5.4",
+    "chai": "^4.3.4",
```

Comment:
We usually use mocha , not sure why you added another package here, please explain ?

Reply:
I copied transactions tests from loopback-cinnector without any changes but it uses chai. I hope it is acceptable.

Reply:
https://github.com/strongloop/loopback-connector/blob/master/test/transaction.test.js

Reply:
Can you help us standardize it here?. I double checked and connectors (ie: mysql, postgresql and this mongodb) already have mocha in the package.json so I guess it should work with it.   Thanks for your patience , this is a great addition to the mongo DB connector.

Reply:
I replaced expect with should in tests